Source/BoundaryConditions/WarpX_PEC.H

```diff
@@ -21,13 +21,13 @@ namespace PEC {
      * \param[in,out] Efield              Boundary values of tangential Efield are set to zero
      * \param[in]     field_boundary_lo   Boundary types of the "low" boundaries
      * \param[in]     field_boundary_hi   Boundary types of the "high" boundaries
+     * \param[in]     bc_type             TODO
```

Comment:
@dpgrote 

I think you added this argument in #5628. Do you have a suggestion for a good docstring? Otherwise I will just go with a trivial `Boundary conditions type`, but I don't find it would be clearly distinguished from the docstrings for `field_boundary_lo` and `field_boundary_hi`.

Reply:
```suggestion
     * \param[in]     bc_type             Boundary condition type to be applied
```
Something for another PR perhaps is to change the name of this routine since it can apply either `PEC` or `PMC` boundary condition..

Reply:
Thanks!